src/labeler.ts

```diff
@@ -40,6 +40,23 @@ export async function run() {
 
     const labels: string[] = [];
     const labelsToRemove: string[] = [];
+
+    if (!changedFiles.length) {
+      core.debug(`exiting early because pr #${prNumber} has no changed files.`);
+
+      if (syncLabels) {
```

Comment:
There is no need to remove labels, because early exit doesn't allow them to appear.